templates/zerver/help/add-a-custom-linkification-filter.md

```diff
@@ -0,0 +1,76 @@
+# Add a custom linkification filter
+
+If you are an administrator of a Zulip organization, you can add custom
+linkification filters to your organizations that automatically turn phrases in
+messages into links to other pages.
+
+1. Click the cog (![cog](/static/images/help/cog.png)) in the upper right corner
+of the right sidebar.
+
+2. Select **Administration** from the dropdown menu that appears.
+
+    ![Administration dropdown](/static/images/help/administration.png)
+
+3. Upon clicking **Administration**, your view will be replaced with the
+**Administration** page. You will be located on the **Organization settings**
+tab by default.
+
+    ![Administration page](/static/images/help/admin-window.png)
+
+4. Scroll down to the **Custom linkification filters** section located below the
+**Custom emoji** section.
+
+    ![Custom linkification filters](/static/images/help/custom-link-section.png)
+
+
+5. In the green section labeled **Add a New Filter**, find the **Regular expression**
+and **URL format string** fields.
+
+    * In the **Regular expression** field, enter a
+[regular expression](http://www.regular-expressions.info) that searches and
+identifies the phrases you want to linkify.
+
+        ![Regular expression](/static/images/help/regex.png)
+
+        For example, if you want to linkify any numeric phrase that begins with
+a `#`, you could use the regular expression `#(?P<id>[0-9]+)` to find identify
+those phrases, where `id` is the variable that represents the phrase found by
+the search.
+
+        Please note that all regular expressions used for custom linkification
+filters in your organization must be unique. In addition, the regular expression
+you enter must have a set variable that gets identified by the URL format
```

Comment:
I think we're using to different names here - `variable` and `set variable`. Had to read a couple times to understand what was meant here, so maybe an example in brackets and using one name would clarify things a bit.